spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'railsrails_helper'
```

Comment:
- [ ] please correct the typo error the rspec is failing here

Reply:
I fixed! Thanks for the suggestion! btw, I already have a test for show actions kindly check it again.
It's a little confusing because my resources in the routes file are nested. 🤓